.gitignore

```diff
@@ -4,3 +4,4 @@ release/
 *.sha256
 *.xz
 *.zip
+/.vs
```

Comment:
A `.gitignore` is actually not a bad idea.  You could add `*.gz` too.

Reply:
*.gz is already present in the .gitignore, in line 3

Reply:
Sorry, `*.tgz`.